eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventData.java

```diff
@@ -132,7 +188,7 @@ public EventData addProperty(String key, Object value) {
      * @return ByteBuffer representing the data.
      */
     public ByteBuffer data() {
-        return data;
+        return body;
     }
```

Comment:
Is there a naming disconnect here between 'data' and 'body'?

Reply:
I forgot to rename this. The property should be "body". Thanks for the catch.